File: src/main/java/wang/zhongpin/pi/model/weather/Coord.java
package wang.zhongpin.pi.model.weather;

public class Coord {

    private double lon;
    private double lat;

    public Coord(double lon, double lat) {
        this.lon = lon;
        this.lat = lat;
    }

    public void setLat(double lat) {
        this.lat = lat;
    }

    public void setLon(double lon) {
        this.lon = lon;
    }

    public double getLat() {
        return lat;
    }

    public double getLon() {
        return lon;
    }
}

File: piClient/src/app/model/weather/coord.ts
export class Coord {
  lon: number;
  lat: number;

  public getlon(): number {
    return this.lon;
  }

  public setlon(value: number): void {
    this.lon = value;
  }

  public getlat(): number {
    return this.lat;
  }

  public setlat(value: number): void {
    this.lat = value;
  }
}
